.env.example

```diff
@@ -1,5 +1,5 @@
 PYTHONPATH=$(pwd):$(pwd)/common:$(pwd)/login:$(pwd)/game
-MONGO_URI=mongo
+MONGO_URI=localhost
```

Comment:
maybe there should be another file for docker
like `.env.docker.example`

Reply:
with only one change mongo/localhost or in docker everything can be pre-configured and there is no need in post fix .example?

Reply:
Hm, lets make it `.env.docker`

Reply:
it's seems out of this scope of this pull request